scripts/aws/setupdb.sh

```diff
@@ -14,3 +14,11 @@ psql -c "CREATE EXTENSION IF NOT EXISTS postgis;"
 
 # Run migrations
 envdir /etc/mmw.d/env /opt/app/manage.py migrate
+
+# Fetch boundary layer sql files
+FILE_HOST="http://santacruz.internal.azavea.com:9999"
+FILES=("boundary_huc12.sql.gz" "boundary_huc10.sql.gz" "boundary_huc08.sql.gz")
```

Comment:
It looks like the `mmw` username is embedded in these but the username on RDS is `modelmywatershed` because `mmw` is too short.